---
title: “Exit Try”只能出现在“Try”语句内
ms.date: 07/20/2015
f1_keywords:
- vbc30393
- bc30393
helpviewer_keywords:
- BC30393
ms.assetid: b8651df3-a32f-478c-a6d8-aa0ef584155f
ms.openlocfilehash: 5280b041f1219ab9577e684c0814c13821ed9de0
ms.sourcegitcommit: f8c270376ed905f6a8896ce0fe25b4f4b38ff498
ms.translationtype: MT
ms.contentlocale: zh-CN
ms.lasthandoff: 06/04/2020
ms.locfileid: "84397111"
---
# <a name="exit-try-can-only-appear-inside-a-try-statement"></a>“Exit Try”只能出现在“Try”语句内
`Exit Try` 只能出现在 `Try` 块语句内。 有冗余的 `Exit Try` 语句，或者 `Exit Try` 语句出现在其对应 `Try` 块的边界之外。  
  
 **错误 ID：** BC30393  
  
## <a name="to-correct-this-error"></a>更正此错误  
  
1. 找到并删除不必要的 `Exit Try` 语句。  
  
2. 将 `Exit Try` 语句移动到代码中适当的位置。  
  
## <a name="see-also"></a>另请参阅

- [Try...Catch...Finally 语句](../language-reference/statements/try-catch-finally-statement.md)
